.github/workflows/build_and_test.yml

```diff
@@ -89,6 +89,8 @@ jobs:
         with:
           crate: cargo-hack
           version: "=0.5.28"
+      - name: Use older version of regex
+        run: cargo update -p regex --precise 1.9.3
```

Comment:
This seems unrelated to this PR and not somethings we'd expected our users to do.
Do we really want that in CI?

Reply:
(Of course, you are right that this change is unrelated to this PR. It's just that this PR's pipeline was the first to be built with the regex 1.10.0 and trigger the problem. If you like, I can move this change to a separate PR.)

Reply:
As long as it only impact the MSRV checks, it makes sense 👍 
Thank you for the explanation.